tests/projects/external-kit/build/rpms/core-kit/kit-metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "bottlerocket-test-kit",
+  "version": "1.1.0",
+  "bottlerocket_sdk": {
+    "name": "bottlerocket-sdk",
+    "version": "0.41.0",
+    "vendor": "bottlerocket"
+  },
+  "dependencies": [
+    {
+      "name": "bottlerocket-core-kit",
+      "version": "1.0.0",
+      "vendor": "bottlerocket"
+    }
+  ]
+}
```

Comment:
I know I'm not the only one around here triggered by missing newlines at the ends of files, so I'll take one for the team and be the one to point it out this time.